deploy/ansible/provisioning.yml

```diff
@@ -81,45 +102,69 @@
         name: iroha_postgres
         image: postgres:9.5
         state: started
+        recreate: yes
         restart: yes
         env:
-          IROHA_POSTGRES_USER: '{{ user }}'
-          IROHA_POSTGRES_PASSWORD: '{{ postgres_passwd }}'
+          POSTGRES_USER: '{{ postgres_user }}'
+          POSTGRES_PASSWORD: '{{ postgres_passwd }}'
         networks:
           - name: iroha_network
-        volumes:
-          - /opt/iroha/data/postgres:/var/lib/postgresql/data
 
     - name: Run redis in docker
       docker_container:
         name: iroha_redis
         image: redis:3.2.8
         state: started
+        recreate: yes
         restart: yes
         networks:
           - name: iroha_network
 
+    - name: Make data dir
+      file:
+        path: /data
+        state: directory
+        mode: 0755
+        owner: iroha
+
+      become: yes
+
+    - name: Copy peers list
+      copy:
+        src: '{{playbook_dir}}/data/peers.list'
+        dest: /data/peers.list
+
+
+    - name: Copy config generator
+      copy:
+        src: '{{playbook_dir}}/generate_config.py'
+        dest: /data/generate_config.py
+
+    - name: Generate config
+      command: python3 /data/generate_config.py -p '{{ postgres_passwd }}' -u '{{ postgres_user }}'
 
     - name: Run iroha in docker
       docker_container:
         name: iroha
-        image: hyperledger/iroha-docker
+        image: hyperledger/iroha-docker:test
         state: started
         restart: yes
+        recreate: yes
+        pull: yes
         ports:
           - "50051:50051"
+          - "10001:10001"
```

Comment:
Why is it here? I though it should be internal only

Reply:
Fyodor approves